truss/tests/test_config.py

```diff
@@ -199,3 +201,36 @@ def test_non_default_train():
     new_config["train"] = updated_train
 
     assert new_config == config.to_dict(verbose=False)
+
+
+def test_huggingface_cache():
+    config = TrussConfig(
```

Comment:
I think it's best for each of these to be a different test

also, can you test multiple `HuggingFaceModel`s here?